.eslintrc

```diff
@@ -0,0 +1,46 @@
+{
+  "env": {
+    "browser": true
+  },
+  "extends": "eslint:recommended",
+  "globals": {
+    "jQuery": true,
```

Comment:
I should have caught this before... but these globals don't make sense for this repo. Can you remove and see what globals eslint complains about (and add those?).